modulefiles/compiler/compilerName/compilerVersion/gftl-shared/gftl-shared.lua

```diff
@@ -0,0 +1,24 @@
+help([[
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+local hierA        = hierarchyA(pkgNameVer,1)
+local compNameVer  = hierA[1]
+local compNameVerD = compNameVer:gsub("/","-")
+
+conflict(pkgName)
+
+local opt = os.getenv("HPC_OPT") or os.getenv("OPT") or "/opt/modules"
+
+local base = pathJoin(opt,compNameVerD,pkgName,pkgVersion)
+
+setenv("GFTL_SHARED_ROOT", base)
+setenv("GFTL_ROOT", base)
```

Comment:
The package is GFTL-Shared, what is GFTL? Is there a `non-shared` version of this library? If so, this can be misleading.

Reply:
I didn't understand this either.

There's gftl-shared: https://github.com/Goddard-Fortran-Ecosystem/gFTL-shared

And gFTL: https://github.com/Goddard-Fortran-Ecosystem/gFTL

I think gftl-shared contains concrete containers of the most common types you would encounter like 32-bit, 64-bit (real, int, etc), while gFTL contains the templated libraries.

Reply:
That's correct. gFTL only provides template files required by gFTL-shared, which includes gFTL as submodule.